packages/desktop-client/src/components/reports/spreadsheets/summary-spreadsheet.ts

```
@@ -0,0 +1,213 @@
+          { count: { $count: '*' } },
+        ]);
+
+    let query = makeRootQuery();
+
+    if (summaryContent.type === 'avgPerMonth') {
+      query = query.groupBy(['date']);
+    }
+
+    const data = await runQuery(query);
+
+    debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)